app/assets/javascripts/SourceCodeGlower/SourceCodeLineAnnotations.js

```diff
@@ -31,9 +31,9 @@ SourceCodeLineAnnotations.prototype.getAnnotationTextManager = function() {
 SourceCodeLineAnnotations.prototype.getAnnotationTextDisplayer = function() {
   return this.annotation_text_displayer;
 }
-
 // Annotate a single Source Code Line
-SourceCodeLineAnnotations.prototype.annotateLine = function(annotation_id, line_num, annotation_text_id) {
+SourceCodeLineAnnotations.prototype.annotateLine = function(
+    annotation_id, line_num, column_start, column_end, annotation_text_id) {
```

Comment:
Identifier 'annotation_id' is not in camel case.<br>Identifier 'line_num' is not in camel case.<br>Identifier 'column_start' is not in camel case.<br>Identifier 'column_end' is not in camel case.<br>Identifier 'annotation_text_id' is not in camel case.